apigentools/commands/init.py

```diff
@@ -16,6 +16,54 @@
 
 
 class InitCommand(Command):
+    def __init__(self, config, args):
+        self.config_file_json = {
```

Comment:
The indentation seems a bit off here. Basically all the entries of the dicts should be indented by one level compared to the variable definition line, e.g.
```python
mydict = {
    "foo": {
        "bar": "baz",
    },
}
```